src/networking.c

```diff
@@ -2821,6 +2821,10 @@ void *IOThreadMain(void *myid) {
     /* The ID is the thread number (from 0 to server.iothreads_num-1), and is
      * used by the thread to just manipulate a single sub-array of clients. */
     long id = (unsigned long)myid;
+    char thdname[16];
+
+    snprintf(thdname, sizeof(thdname), "io_thd_%ld", id);
+    pthread_setname_np(pthread_self(), thdname);
```

Comment:
FYI on macOS ther first parameter (pthread_self) shouldn't be present.

https://stackoverflow.com/questions/2057960/how-to-set-a-threadname-in-macosx

Reply:
OK, I'll fix this later.